app/views/people/_biography.html.erb

```diff
@@ -1,4 +1,4 @@
-<section id="biography" class="govuk-!-padding-bottom-9">
+<section id="biography" class="govuk-!-padding-bottom-9" dir="<%= page_text_direction %>" lang="<%= person.locale %>">
```

Comment:
Same here - this shouldn't have right to left since it contains text that won't be translated.